pkg/web/apps/apps_http_handler.go

```diff
@@ -92,7 +92,7 @@ func (a *httpHandler) UpdateAppVersions(c echo.Context) error {
 		return httperrors.GetHTTPResponseFromErr(c, errUpdate)
 	}
 
-	return c.JSON(http.StatusOK, "")
```

Comment:
Should be

```go
return c.NoContent(http.StatusNoContent)
```

otherwise you are returning an empty string (which is content).

Reply:
It is really something that will no make any diff for us. However, as you wish it is changed.